### PR TITLE
Rajab month

### PR DESCRIPTION
**Pull request by fzerorubigd**
The job is done by @neda1985 , just pushed by me.

### CODE REVIEW

File: hijri/07-rajab.yml [hunk 1]
```diff
@@ -1 +1,147 @@
...
+  title:
+    fa_IR:  ولادت امام محمد باقر، پنجمین امام شیعه دوازده‌امامی
+  description:
+    fa_IR:  ولادت امام محمد باقر، پنجمین امام شیعه دوازده‌امامی
+  year: 57
+  month: 7
+  day: 1
+  calendar:
+    en_US:
+      - Shea
```
Comment (okian) [inline hijri/07-rajab.yml]: Shia

File: hijri/07-rajab.yml [hunk 2]
```diff
@@ -1 +1,147 @@
...
+  title:
+    fa_IR: درگذشت علی النقی
+  description:
+    fa_IR: درگذشت علی النقی، دهمین امام شیعه دوازده‌امامی
+  year: 254
+  month: 7
+  day: 3
+  calendar:
+    en_US:
+      - Shea
```
Comment (okian) [inline hijri/07-rajab.yml]: Shia

File: hijri/07-rajab.yml [hunk 3]
```diff
@@ -1 +1,147 @@
...
+  title:
+    fa_IR:  ولادت محمد تقی
+  description:
+    fa_IR: ولادت محمد تقی، نهمین امام شیعه دوازده‌امامی
+  year: 195
+  month: 7
+  day: 10
+  calendar:
+    en_US:
+      - Shea
```
Comment (okian) [inline hijri/07-rajab.yml]: Shia

File: hijri/07-rajab.yml [hunk 4]
```diff
@@ -1 +1,147 @@
...
+- partial_key: "imam_ali_bd"
+  title:
+    fa_IR: ولادت علی بن ابی‌طالب
+  description:
+    fa_IR:  ولادت علی بن ابی‌طالب، نخستین امام شیعیان
+  month: 7
+  day: 13
+  calendar:
+    en_US:
+      - Shea
```
Comment (okian) [inline hijri/07-rajab.yml]: Shia

File: hijri/07-rajab.yml [hunk 5]
```diff
@@ -1 +1,147 @@
...
+  title:
+    fa_IR: درگذشت موسی کاظم
+  description:
+    fa_IR: درگذشت موسی کاظم، هفتمین امام شیعه دوازده‌امامی
+  year: 183
+  month: 7
+  day: 25
+  calendar:
+    en_US:
+      - Shea
```
Comment (okian) [inline hijri/07-rajab.yml]: Shia

File: hijri/07-rajab.yml [hunk 6]
```diff
@@ -1 +1,147 @@
...
+- partial_key: "muhammad_first_revelation"
+  title:
+    fa_IR: عید مبعث
+  description:
+    fa_IR: عید مبعث، سالروز بعثت پیامبر اسلام در جهان اسلام
+  month: 7
+  day: 27
+  calendar:
+    en_US:
+      - Shea
```
Comment (okian) [inline hijri/07-rajab.yml]: Shia

**Comment by fzerorubigd** [comment]
duplicate of #98

**Pull request closed without merging**